randovania/games/planets_zebeth/logic_database/Brinstar.txt

```diff
@@ -276,29 +271,22 @@ Extra - map_name: Brinstar/x312_y30
 > Door to Corridor 3; Heals? False
   * Layers: default
   * Normal Door to Corridor 3/Door to Corridor-connecting room
-  > Bridge
+  > Door to Varia Suit corridor
+      Hi-Jump Boots
+  > Door to Corridor 7
       Trivial
 
 > Door to Varia Suit corridor; Heals? False
   * Layers: default
   * Normal Door to Varia Suit corridor/Door to Corridor-connecting room
-  > Bridge
-      Trivial
+  > Door to Corridor 3
+      Can Use Bombs
```

Comment:
my nestroid knowledge isnt perfect, are there not fake blocks/crumble blocks that allow you to fall down?

Reply:
I cannot find a way to leave that room without returning through the shot/bomb blocks.

Reply:
> my nestroid knowledge isnt perfect, are there not fake blocks/crumble blocks that allow you to fall down?

no the only way to go down those blocks is bomb